client/lib/accounts/config.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Useraccounts configuration.
+ *
+ */
+AccountsTemplates.configure({
+    defaultLayout: 'layout',
+    defaultLayoutRegions: {
+        header: 'header'//,
+        //footer: '_footer'
+    },
+    defaultContentRegion: 'main',
+    showForgotPasswordLink: true,
+    overrideLoginErrors: true,
+    enablePasswordChange: true,
+    sendVerificationEmail: true,
```

Comment:
Right now, it will just try to send a verification mail, but nothing will actually send untill we doesn't have mailer configured. It can be added afterwards.

Get a look at this: https://github.com/michaltakac/meteor-boilerplate/blob/master/server/email/config.js
...and email templates here: https://github.com/michaltakac/meteor-boilerplate/blob/master/server/templates/emails/contactEmail.handlebars

^ Both added in `/server` folder.

Package used for implementing this ^:
 - cmather:handlebars-server@0.2.0
 - cunneen:mailgun

Mailgun configuration (username, password) should be added to settings.json and shouldn't be commited. Example settings could look like:

`settings.example.json`:
```json
{
  "public": {
    
  },
  "private": {
    "mailgun": {
      "username": "xxxxxxxxxx",
      "password": "xxxxxxxxxxxxxxxx"
    }
  }
}
```